tools/monorepo-utils/src/slack/commands/slack/slack-message.ts

```diff
@@ -53,7 +53,10 @@ export const slackMessageCommand = new Command( 'message' )
 			try {
 				const { statusCode, body } = await requestAsync(
 					options,
-					JSON.stringify( { channel, text } )
+					JSON.stringify( {
+						channel,
+						text: text.replace( /\\n/g, '\n' ),
```

Comment:
Because of [how bash interprets newline characters](https://stackoverflow.com/questions/8467424/echo-newline-in-bash-prints-literal-n/8467448#8467448) I needed to fix a bug here. Bash interprets `\n` as the characters `\` and `\n` so we have to replace them with the newline character using JS.